generation/WinSDK/enums.json

```diff
@@ -30873,5 +30893,66 @@
         "field": "bV4V4Compression"
       }
     ]
+  },
+  {
+    "type": "uint",
+    "name": "MODIFIERKEYS_FLAGS",
+    "flags": "true",
+    "autoPopulate": {
+      "filter": "MK_",
+      "header": "WinUser.h"
+    },
+    "members": [],
+    "uses": [
+      {
+        "method": "IRichEditOleCallback::GetDragDropEffect",
+        "parameter": "grfKeyState"
+      },
+      {
+        "method": "IDropSource::QueryContinueDrag",
+        "parameter": "grfKeyState"
+      },
+      {
+        "method": "IDropTarget::DragOver",
+        "parameter": "grfKeyState"
+      },
+      {
+        "method": "IDropTarget::DragEnter",
+        "parameter": "grfKeyState"
+      },
+      {
+        "method": "IDropTarget::Drop",
+        "parameter": "grfKeyState"
+      }
+    ]
+  },
+  {
+    "type": "int",
```

Comment:
These should be uint because they're used in CreateWindowEx's dwStyle, which is a DWORD.